clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -451,11 +443,11 @@ private CompletableFuture<Map<TopicPartition, OffsetAndMetadata>> addOffsetFetch
         /**
          * Clear {@code unsentOffsetCommits} and moves all the sendable request in {@code unsentOffsetFetches} to the
          * {@code inflightOffsetFetches} to bookkeep all of the inflight requests.
-         *
+         * <p>
```

Comment:
was the intention here to add an extra line in-between? should be <p/> if so

Reply:
😓 - must be the residual from some edits..